Add compatible node engine version to package.json

jeremymeng opened the pull request:
This version of `~0.10.22` is taken from https://github.com/Azure-Samples/app-service-api-node-contact-list/blob/6664e24a2b19dd4a48e88ba0345ed0ff14cd4c80/end/ContactList/package.json#L16.

azurecla commented:
Hi __@arunvinoth__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

jeremymeng committed:
Add compatible node engine version to package.json

This version of `~0.10.22` is taken from https://github.com/Azure-Samples/app-service-api-node-contact-list/blob/6664e24a2b19dd4a48e88ba0345ed0ff14cd4c80/end/ContactList/package.json#L16.

GitHubber17 committed:
Update app-service-web-tutorial-rest-api.md
